src/index.ts

```diff
@@ -104,6 +104,7 @@ export { default as PlaySettings } from './suppliers/model/PlaySettings';
 export { default as Play } from './plays/model/Play';
 export { default as PlayScheduleConfiguration } from './plays/model/PlayScheduleConfiguration';
 export { default as PlayUpdateRequest } from './plays/model/PlayUpdateRequest';
+export { default as PlayCreateRequest } from "./plays/model/PlayCreateRequest";
```

Comment:
@appu-channelape Can you change this single quote like the build server is suggesting?